tests/conftest.py

```diff
@@ -28,10 +28,23 @@ def _method(zip_name):
 
         print("\nRemoving previous gear...")
 
-        if Path(FWV0 / "config.json").exists():
-            Path(FWV0 / "config.json").unlink()
-
-        for dir_name in ["input", "output", "work", "freesurfer", "templateflow"]:
+        for file_name in [
+            "config.json",
+            "time_output.txt",
+        ]:
+            if Path(FWV0 / file_name).exists():
+                Path(FWV0 / file_name).unlink()
+
+        for dir_name in [
+            "input",
+            "output",
+            "work",
+            "freesurfer",
+            "templateflow",
+            "unzip-dir1",
```

Comment:
Would change, if name changes in run.py.

Reply:
Good catch.  Changed also.